docs/content/uijson_format/index.rst

```diff
@@ -4,31 +4,100 @@ UI.JSON Format
 About
 ^^^^^
 
-The **ui.json** format provides a User Interface (UI) between geoh5py and `Geoscience ANALYST Pro
-<http://www.mirageoscience.com/our-products/software-product/geoscience-analyst>`_. The file structure is built on an array of `JSON objects <https://json-schema.org/specification.html>`_, each representing a parameter that is used in a python script. An object contains members that control the style and behaviour of the UI. In general only a **label** and **value** member is required in each object, however as outlined below, there are many types of input and dependencies that can be drawn on throughout the file. On output from Geoscience ANALYST, the value and whether the parameter is enabled will be updated or added to each JSON. Extra objects in the JSON are allowed and are ignored, but written out by Geoscience ANALYST. In general, objects will be put in order that they are set in the JSON. The exception is data parameters that depend on object parameters. Placing those parameters in the same group will ensure that they are close in the UI.
+The **ui.json** format provides a schema to create a simple User Interface (UI) between geoh5py and `Geoscience ANALYST Pro
+<http://www.mirageoscience.com/our-products/software-product/geoscience-analyst>`_. The format uses JSON objects to represent `script parameters <./json_objects.rst>`_ used in the UI, and pass those parameters to an accompanying python script.
 
 
-Input Objects
-^^^^^^^^^^^^^
-Within the **ui.json** file, each JSON object with **value** and **label** members will be considered a parameter to the UI. The following JSON objects could also be present:
+Each ui.json object requires at least a **label** and **value** member, however additional members can be used to define different types of input and additional dependencies between parameters.
+
+For example, a simple ui.json below describes a single parameter called 'grid_object', which is used to select a block model within a geoh5 file.
+
+.. code-block:: json
+
+    {
+    "grid_object": {
+    "meshType": ["{B020A277-90E2-4CD7-84D6-612EE3F25051}"],
+    "main": true,
+    "label": "Select Block Model",
+    "value": ""
+    }
+    }
+
+Note: The **meshType** used to select the grid object is defined by a list of UUID. A complete list of UUID's for geoh5 object types are available in the `geoh5 objects documentation <../content/geoh5_format/analyst/objects.rst>`_.
+
+
+Usage with Geoscience ANALYST Pro
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A ui.json file shall contain the parameters that reference a python script to run. Ensure the python path is set up correctly in Geoscience ANALYST preferences. Optionally include a conda environment to use in the preferences and in the ui.json script parameter **conda_environment**.
```

Comment:
Consider sticking to an active verb tense rather than switching to imperative. Slightly friendlier/neutral tone.